packages/client/src/components/app/blocks/form/InnerFormBlock.svelte

```diff
@@ -198,22 +211,19 @@
           {/if}
         </BlockComponent>
       {/if}
-      <BlockComponent type="fieldgroup" props={{ labelPosition }} order={1}>
-        {#each fields as field, idx}
-          {#if getComponentForField(field.name)}
-            <BlockComponent
-              type={getComponentForField(field.name)}
-              props={{
-                validation: field.validation,
-                field: field.name,
-                label: field.displayName,
-                placeholder: field.displayName,
-              }}
-              order={idx}
-            />
-          {/if}
-        {/each}
-      </BlockComponent>
+      {#key fields}
```

Comment:
Do we need to key this whole thing? This will mean every field will remount whenever any field setting changes. We could maybe instead add in the keying param to the `#each` loop, depending on what property requires remounting - e.g. `{#each fields as field, idx (field._component}` or whichever property it is that requires remounting.